organization/src/main/openapi/organization.yaml

```diff
@@ -14,7 +14,7 @@ tags:
 paths:
   /organizations:
     post:
-      operationId: registerOrganization
+      operationId: createOrganization
       tags:
         - organization
       summary: Register a new organization
```

Comment:
The summaries should probably be updated accordingly too. Here it still mentions "registering" an organization. Same for delete/unregister and maybe others.